x/gov/init.go

```diff
@@ -72,27 +72,35 @@ func (*Initializer) FromGenesis(opts weave.Options, params weave.GenesisParams,
 
 	// handle election rules
 	rulesBucket := NewElectionRulesBucket()
+	rulesBucketSeq := rulesBucket.Sequence(electionRuleSequence)
 	for i, r := range governance.Rules {
 		electorateID := encodeSequence(r.ElectorateID)
 		_, _, err := electBucket.GetLatestVersion(kv, electorateID)
 		if err != nil {
 			return errors.Wrapf(err, "failed to load electorate with id: %d", r.ElectorateID)
 		}
+		newRuleID, err := rulesBucketSeq.NextVal(kv)
+		if err != nil {
+			return errors.Wrap(err, "unable to generate ElectionRule sequence")
+		}
+
 		rule := ElectionRule{
 			Metadata:     &weave.Metadata{Schema: 1},
 			Admin:        r.Admin,
 			Title:        r.Title,
 			VotingPeriod: r.VotingPeriod,
 			Threshold:    Fraction{Numerator: r.Threshold.Numerator, Denominator: r.Threshold.Denominator},
 			ElectorateID: electorateID,
+			Address: weave.Condition(newRuleID).Address(),
```

Comment:
req: you can not cast to condition but create a new one via `NewCondition` with [`gov/rule/newRuleID`](https://github.com/iov-one/devnet-operations/blob/master/manifests/weave/davenet/bns/genesis.json#L136)